src/collections/news/2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush/index.mdx

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+type: news
+author: the Newsroom
+date: 2019-12-27 08:04:00 +0000
+title: "Vendors make a splash in 2019 service mesh implementation rush"
+image: ./2019-12-27-vendors-make-a-splash-in-2019-service-mesh-implementation-rush.png
+#permalink: https://searchapparchitecture.techtarget.com/feature/Vendors-make-a-splash-in-2019-service-mesh-implementation-rush
```

Comment:
Remove this

Reply:
So I should remove all selected and not put anything there right? Just to be very clear

Reply:
Yes